requestor-tutorials/service-development/service-example-2-managed-erigon.md

```diff
@@ -1,4 +1,63 @@
+---
+description: Ethereum client as a service, implemented as a self-contained runtime
```

Comment:
just had a thought that it's kinda awkward to use "as a" twice in the same sentence ... so maybe e.g. "using" ? or "with the use of" ... 

```suggestion
description: Ethereum client as a service, implemented using a self-contained runtime
```

Reply:
I'm not entirely convinced to this suggestion ... just couldn't think of anything better quickly